open-apparel-registry/open-apparel-registry
# Add endpoints for info website

## Overview

In the info website relaunch, there will be 3 stats listed on the home
page about the OAR: facilities count, active contributors count, and
count of countries with active facilities. In order for those numbers
to automatically update as the OAR updates, we have added endpoints to
provide the active contributors count and the active countries count.

Connects #1476 

## Demo
<img width="979" alt="Screen Shot 2021-10-12 at 10 26 39 AM" src="https://user-images.githubusercontent.com/21046714/136981455-38a14ec8-2f71-468c-bbb1-3fc265e9bb42.png">
<img width="980" alt="Screen Shot 2021-10-12 at 10 26 26 AM" src="https://user-images.githubusercontent.com/21046714/136981423-554c8736-4cfe-427c-a020-2a677057e7b1.png">

## Notes

Note that the facilities count, the third endpoint required for the info website, already existed and so didn't need to be created for this task.

## Testing Instructions

* Visit the homepage and confirm the contributors in the search dropdown haven't changed. 
* Visit the api at http://localhost:8081/api/docs/#!/contributors/contributors_active_count_list and confirm the response matches the number of contributors returned from http://localhost:8081/api/docs/#!/contributors/contributors_list.
* In the dbshell run `select count(distinct country_code) from api_facilityindex;` and confirm this number matches the count returned from http://localhost:8081/api/docs/#!/countries/countries_active_count_list.

## Checklist
